src/components/Calculator.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+class Calculator extends React.Component { // eslint-disable line
```

Comment:
- [ ] You are not supposed to disable Linters. There are great tools that flag programming errors. You need to fix these errors so that your code should be clean.
- **Hint:** You may probably be struggling with Linters since you are extending `React.component.`. I'll kindly advise you to try out `React.PureComponent`, that's

 ```.js 
      class Calculator extends React.PureComponent {
 ```